spec/requests/users_controller_spec.rb

```diff
@@ -31,6 +33,26 @@
         post api_v1_users_path, params: valid_user_params, headers: headers
         expect(User.find_by(email: valid_user_params[:user][:email]).role).to eq(role)
       end
+
+      context 'User language' do
+        before do
+          allow(I18n).to receive(:default_locale).and_return(:gl)
+        end
+
+        it 'Persists the user language in the user record' do
+          expect { post api_v1_users_path, params: valid_user_params, headers: }.to change(User, :count).from(0).to(1)
+          expect(response).to have_http_status(:created)
+          expect(User.take.language).to eq('en')
```

Comment:
Just for extra safety, I would do `expect(User.find_by(email: valid_user_params[:user][:email]).language).to eq('en')`